packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -365,6 +365,8 @@ export class GarbageCollector implements IGarbageCollector {
     public readonly trackGCState: boolean;
 
     private readonly testMode: boolean;
+    private readonly sweepV0TimeoutMs: number | undefined;
```

Comment:
Not sure I see a value in this member, given that value from feature gate overwrites this.sweepTimeoutMs and that's how it gets used.

Reply:
Ah, did not notice sweepV0. Maybe break it into more targeted properties, like deleteUnusedRoutesImmidiatly() (to be used in sweepIfEnabled()